backend.rb

```diff
@@ -51,12 +55,16 @@ class Backend < Sinatra::Base
     authorize_payment_response = HTTParty.post(authorize_url, authorize_request_options)
 
     if (authorize_payment_response && authorize_payment_response.code == 201)
+      # The purchase was authorized (and has essentially been created as a
+      # resource available at the location specified in the authorization
+      # requests's response). Capture it.
```

Comment:
That's something that should be available at developers.klarna.com at some point. It's the APIs documentation, and therefor not the best fit for anything we have on github.